_pkgdown.yml

```diff
@@ -1,18 +1,39 @@
 url: epidist.epinowcast.org/
 template:
-  bootstrap: 5
-  bootswatch: flatly
-  theme: monokai
+  package: enwtheme
   twitter:
     creator: "@seabbs"
```

Comment:
This is boiler plate from Epinowcast that we probably don't need here

Reply:
I don't fully understand the interface here as to what (which lines) this comment is pointing at?